vars/buildWDKAutoSwitch.groovy

```diff
@@ -146,6 +149,16 @@ def call(Map config = [unityVersions:[]]) {
                     }
                   }
 
+                  def catchStep(Exception e) = {
+                    echo "Failed to build required Unity version ${version}\n${e}"
```

Comment:
I don't think we have to reprint the error. Which might also only contain internal things. Jenkins will already log the failing message.

Reply:
I see.